src/styles/settings.css

```diff
@@ -18,267 +33,396 @@ body {
+input[type="time"] {
+  -moz-appearance: textfield;
 }
 
-.subsettings-details {
-  overflow: hidden;
-  height: 0px;
-  margin-left: 10px;
-  transition: height 300ms ease-in-out;
+input::-webkit-inner-spin-button,
+input::-webkit-outer-spin-button
+{
+  -webkit-appearance: none;
 }
 
-.subsettings-details span {
-  display: block;
+input[type="number"].xs {
+    width: 60px;
 }
 
-.list {
+input[type="range"] {
+  -webkit-appearance: none;
+  height: var(--checkbox-size);
   padding: 0;
+  background: var(--main-bg-color);
+  border: 1px solid var(--border-color);
+  outline: none;
+  -webkit-transition: .2s;
+  transition: all .2s;
+}
+
+input[type="range"]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: var(--checkbox-size);
+  height: var(--checkbox-size);
+  border-radius: 50%; 
+  background: var(--selected-color);
+  cursor: pointer;
+}
+
+input[type="range"]::-moz-range-thumb {
+  width: var(--checkbox-size);
+  height: var(--checkbox-size);
+  border-radius: 50%;
+  background: var(--selected-color);
+  cursor: pointer;
+}
+
+/* sidebar */
+
+nav {
+  flex: 0 0 auto;
+  background: #323232;
+  color: white;
+  height: 100%;
+  width: var(--sidebar-width);
+  z-index: 10;
+}
+
+nav .logo {
+  display: flex;
+  align-items: center;
+  margin-left: 15px;
+  height: 66px;
+}
+
+nav .section {
+  margin-left: 10px;
+}
+
+nav .section h3, nav .section .setting {
+  font-size: 11px;
+  text-transform: uppercase;
+  color: rgb(123,123,123);
   margin: 0;
-  list-style: none;
+  padding: 19px 0px 7px 8px;
 }
 
-.list li {
-  min-height: 40px;
+nav .section ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
 }
 
-.list li label {
-  font-size: 14px;
-  height: 22px;
-  line-height: 22px;
-  vertical-align: middle;
+nav .section li, nav .section li a {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  height: 32px;
+  width: 160px;
+  border-radius: var(--m-radius);
+  font-size: var(--font-size);
+  font-weight: 400;
+  color: white;
+  text-decoration: none;
+  opacity: 0.8;
 }
 
-input[type='checkbox'] {
-  height: 1.5em;
-  width: 1.5em;
+nav .section li:hover {
+  opacity: 1;
 }
 
-.description {
-  color: rgb(119, 119, 119);
-  padding: 5px;
-  align-self: stretch;
-  width: 100%;
-  padding: 5px 0;
+nav .section li.active {
+  opacity: 1;
+  background: rgb(73,73,73);
+}
+
+nav .section li .nav-icon {
+  height: 32px;
+  width: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-.list li:hover .description {
-  visibility: visible;
+nav .section li span {
+  padding-left: 5px;
+}
+
+/* main content */
+
+div.content {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  overflow: auto;
 }
 
 header {
-  width: 100%;
+  height: 66px;
   display: flex;
-  justify-content: center;
-  box-shadow: inset 0px -1px 1px -1px #bbb;
-  position: relative;
+  align-items: center;
+  flex: 0 0 auto;
+  background: white;
```

Comment:
The dark mode looks amazing bar the header
![image](https://user-images.githubusercontent.com/1716853/69672413-14fa4c80-10be-11ea-9d54-f6cf9e6e7bd9.png)

I think we should add a css-var for it as well `--header-bg-color`